business_requirement_earned_value/i18n/sl.po

```diff
@@ -1,131 +0,0 @@
-# Translation of Odoo Server.
```

Comment:
Why remove a translation, file?

Reply:
it was for a version of the module which is completely different